iothub/service/src/Jobs/ScheduledJob.cs

```diff
@@ -24,24 +24,93 @@ public class ScheduledJob
         [JsonProperty(PropertyName = "queryCondition", NullValueHandling = NullValueHandling.Ignore)]
         public string QueryCondition { get; internal set; }
 
+        /// <summary>
+        /// Scheduled job start time in UTC.
+        /// </summary>
+        [JsonProperty(PropertyName = "createdDateTimeUtc", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
"createdDateTimeUtc" or "CreatedTimeUtc"? seems inconsistent

Reply:
Is this a duplicate of the next property?

Reply:
https://msazure.visualstudio.com/One/_git/Azure-IoT-Hub-Main?path=/swagger/2021-11-30/Service_2021-04-12.json&version=GBmaster&line=2477&lineEnd=2478&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents

Reply:
I've reworked this a little with David